data/assets/examples/renderable/renderablesphereimageonline/simplesphereimageonline.asset

```diff
@@ -0,0 +1,28 @@
+local Radius = 696340000 * 20 -- The radius of the Sun in meters, times 20
+
+local SimpleSphereOnlineImage = {
+  Identifier = "SimpleSphereOnlineImage",
+  Renderable = {
+    Type = "RenderableSphereImageOnline",
+    URL = "http://data.openspaceproject.com/examples/renderableplaneimageonline.jpg",
+
+    -- Parameters from the parent class `RenderableSphere`
+    Size = Radius,
+    Segments = 16
+  },
+  GUI = {
+    Name = "Simple Sphere Online Image",
```

Comment:
```suggestion
    Name = "Sphere - Online Image",
```
? 
Or do we have a reason to call it "Simple" sphere? 

I also suggest removing the word "simple" from the description

Reply:
I think it only makes sense if there are multiple assets where one is using only the required parameters for example. Otherwise, I agree

Reply:
As we discussed during the meeting session - maybe we should discuss this during an upcoming meeting to make sure that we are consistent in naming schemes, etc. 

One suggestion would be to bring it up during our Monday meeting next week. Thoughts @alexanderbock ?

Reply:
Yea, before merging any of these, let's discuss it on Monday and decide on a format